### ISSUE TITLE
Server side rendering

### ISSUE THREAD

**Issue by hdjirdeh** [hdjirdeh/angular2-hn#26]
Because this is a regular Angular app, it doesn't work when JS is disabled. We'll need to look into using [Angular Universal](https://github.com/angular/universal) to see if we can get some parts of the app (if not all) working without any JS  on the browser.

**Comment by housseindjirdeh** [comment]
Now that `platform-server` is part of 4.0.0, now's a great time as ever to take a crack at this :)

Will hopefully get to it this week.

**Comment by MarkPieszak** [comment]
Take a look here where we're working on a starter for it. @housseindjirdeh 
https://github.com/FrozenPandaz/ng-universal-demo

We'll be updating the official angular/universal-starter closer to the 4.0 release.

There are still some minor tweaks that we need to get in.

**Comment by housseindjirdeh** [comment]
Appreciate it! That will definitely come in handy :)

**Comment by paulogr** [comment]
Is ssr still relevant for this app? If so I can help with a PR because I'm just working on this right now in a project. Thanks!

**Comment by housseindjirdeh** [comment]
@paulogr yes this is still relevant! Thank you so much, I would definitely appreciate a PR to get this added. Will you be using a specific library to add this?

Please let me know if I can help in anyway as well! Thanks again mate 🙌🏾
